src/react_native/reanimated.cljs

```diff
@@ -57,9 +57,12 @@
 ;; Easings
 (def bezier (.-bezier ^js Easing))
 
+(def in-out
+  (.-inOut ^js Easing))
+
 (def easings
   {:linear  (bezier 0 0 1 1)
-   :easing1 (bezier 0.25 0.1 0.25 1) ;; TODO(parvesh) - rename easing functions, (design team input)
+   :easing1 (bezier 0.25 0.1 0.25 1) ; TODO(parvesh) - rename easing functions, (design team input)
```

Comment:
Got design input there are no specific names for easing functions, please remove this comment